pagy.gemspec

```diff
@@ -30,12 +23,4 @@ Gem::Specification.new do |s|
   s.add_development_dependency 'kalibera'
   s.add_development_dependency 'memory_profiler'
   s.add_development_dependency 'rubocop', '~> 0.55.0' # needs to be fixed since it constantly adds new cops
-
-
-  s.post_install_message = <<END_OF_MESSAGE
```

Comment:
annoying ... always hate these and most other gems thankfully don't do that

Reply:
The original intention of this message was getting attention on the breaking changes of 0.8.0, and providing a link to the changelog that list them would have been useful. Then the text was too long and I reduced it leaving only the link, and I realize that the original goal is missing now.
A better option would have been something like "Please, check for breaking changes: https://...".
It was meant to be temporary (only for version 0.8.0), but then it would not appear for version after the breaking change and upgrading is not a linear process, so we have to decide whether it is too annoying or somehow useful.
Maybe we should just keep a (better) line of text? Less annoying and still useful?